brigade.js

```diff
@@ -1,9 +1,9 @@
 const { events, Job, Group } = require("brigadier");
 
-const projectOrg = "deislabs";
+const projectOrg = "cnabio";
 const projectName = "duffle";
 
-const goImg = "quay.io/deis/lightweight-docker-go:v0.7.0";
+const goImg = "golang:1.13";
```

Comment:
cc @vdice - I just used the official Go image here - do we want to update original Go image we were using?

Reply:
+1 for using the official go image

Reply:
Official sounds good to me!  I'll test this branch to be sure... can't recall if the official image has `make` for instance...

Reply:
(It does have `make` 🎉 )   The other targets look fine; the buildAndPublishImage job worked for my fork off of this branch.  I didn't explicitly test the github release job, but I don't see why that would break.  The `brigade.js` needs an overhaul to slim down and use utilities from the brigade-utils library... we should file a follow-up.